### PR TITLE
PT-294 Add logger to write down events

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: lib/routes/highwaterApi.js [hunk 1]
```diff
@@ -38,6 +38,24 @@ module.exports = function (userApiClient, kissClient, salt) {
     return hash.finalize().toString().substr(0, len);
   }
 
+  function sendMetrics(req, user, hash = true) {
```
Comment (CedricPerotto) [inline lib/routes/highwaterApi.js:41]: the hash parameter isn't used in the function or it's an omission to hash user param (and maybe the req.params.patientID added).

**Review by CedricPerotto — COMMENTED** [review]
There is no hash on userid field in the file log (see my comment below). I don't know if it's intentional.

File: lib/routes/highwaterApi.js (hunk 1 above)
Comment (franck-fourel) [reply lib/routes/highwaterApi.js:41]: it is intentional but I'm still wondering if I should hash it or not. 
The userId hash is done when sending the data to KissMetrics. 
By the way it's answering my question: if we send the data to Sumologic, we have to hash it.

**Comment by CedricPerotto** [comment]
LGTM

**Pull request merged**